flink-tests/src/test/java/org/apache/flink/test/javaApiOperators/util/CollectionDataSets.java

```diff
@@ -54,6 +51,18 @@
  */
 public class CollectionDataSets {
 
+	public static DataSet<Tuple1<String>> getTupleDateSet(ExecutionEnvironment env) {
+
+		List<Tuple1<String>> data = new ArrayList<>();
+		data.add(new Tuple1<String>("%leo*"));
+		data.add(new Tuple1<String>(("%deng*")));
+		data.add(new Tuple1<String>("%hello*"));
```

Comment:
I think we should add some negative test data (without `%` and `*`).

And I would like not to touch other modules if possible, it's better to move the specific test data to the specific test class.